OpenRA.Game/Graphics/WorldRenderer.cs

```diff
@@ -220,34 +247,44 @@ public void Draw()
 				foreach (var r in g)
 					r.Render(this);
 
+			Game.Renderer.Flush();
+
```

Comment:
A future PR will chop this method in half at this line. Everything above will be drawn into the world framebuffer, everything below as UI.